en/docs/guides/access-delegation/oidc-hybrid-flow.md

```diff
@@ -0,0 +1,227 @@
+    ```
+
+2. Optionally, you can decrypt the `id_token`. It will be similar to the decrypted `id_token` shown below.
+
+    ``` java
+    {
+    “at_hash”: “JrZY9MtYVEIIJUx-DDBmww”,
+    “c_hash”: “XCRuS2hEOB_3HdxoE3JqOg”,
+    “sub”: “admin”,
+    “aud”: [
+    “nczbg5m5xxt6tP4UMZwB6PtQoQoa”
+    ],
+    “azp”: “nczbg5m5xxt6tP4UMZwB6PtQoQoa”,
+    “iss”: “https://localhost:9443/oauth2/token",
+    “exp”: 1510833164,
+    “nonce”: “asd”,
+    “iat”: 1510833160
+    }
+    ```
+
+3. You can send the code to the token endpoint to request for an access token, refresh token and id\_token. You can use the same curl command provided for the code token specified [above](#curl).
+
+    You will receive the following response from the token endpoint. 
+
+    ``` 
+    {“access_token”:”1940a308-d492–3660-a9f8–46723cc582e9",”refresh_token”:”6b96cc3a-00da-3d7d-acd1–5aaf76dcd9d4",”scope”:”openid”,”id_token”:”eyJ4NXQiOiJOVEF4Wm1NeE5ETXlaRGczTVRVMVpHTTBNekV6T0RKaFpXSTRORE5sWkRVMU9HRmtOakZpTVEiLCJraWQiOiJOVEF4Wm1NeE5ETXlaRGczTVRVMVpHTTBNekV6T0RKaFpXSTRORE5sWkRVMU9HRmtOakZpTVEiLCJhbGciOiJSUzI1NiJ9.eyJhdF9oYXNoIjoiSnJaWTlNdFlWRUlJSlV4LUREQm13dyIsInN1YiI6ImFkbWluIiwiYXVkIjpbIm5jemJnNW01eHh0NnRQNFVNWndCNlB0UW9Rb2EiXSwiYXpwIjoibmN6Ymc1bTV4eHQ2dFA0VU1ad0I2UHRRb1FvYSIsImlzcyI6Imh0dHBzOlwvXC9sb2NhbGhvc3Q6OTQ0M1wvb2F1dGgyXC90b2tlbiIsImV4cCI6MTUxMDgzMzMwNywibm9uY2UiOiJhc2QiLCJpYXQiOjE1MTA4MzMzMDN9.k69ufNIJHJHb6foeRSMVoJsgAWz0q65_8R6Lhz-tIW-tdLDI7eNg3kSL5-S2T3uFn7XFvn113wEWvCS8X3JBCIPMAFCmGBCR_L5pCh_OO6_xQeZyfa0fx_R27kZ9EIW5u0WSSjlpzzvr_50YldCfXMhZASjZlA5sCZ9BReyhkEUW_kSCWUDJEPaFQqgKVNfnRmr1q4N2lJwXPHjjE-4BcTMxKY87mqFzq_HVdXc1SRVIG0iuWkiYaD34pK8ZI12GFGSmOpDzhYb06uxrR8GC4jpq_WHMvMKrPrLaoVkEFaqomgxLIOJaNZJzqpe3wlaWM952eTndpSW0HSR5kgZgmw”,”token_type”:”Bearer”,”expires_in”:297198}
+    ```
+
+4. If there are two id\_tokens issued, where one id\_token is from authorization endpoint and other is from token endpoint, be sure to perform the [neccessary validations](#validations), which are based on the OpenID Connect specification.
+
+----
+
+## id_token validations
+
+If there are two id\_tokens issued, where one id\_token is from authorization endpoint and other is from token endpoint, be sure to perform the following validations based on the OpenID Connect specification.
```

Comment:
It's a little odd to have the same sentence as a step in the previous topic and then as the intro here. Shall we remove/modify this?